prisma-fmt/src/lib.rs

```diff
@@ -119,7 +142,39 @@ pub fn get_config(get_config_params: String) -> Result<String, String> {
     get_config::get_config(&get_config_params)
 }
 
-pub fn get_dmmf(get_dmmf_params: String) -> String {
+/// This is the same command as get_config()
```

Comment:
I think this was copy-pasted, or you meant `get_dmmf()` (but which get dmmf?)

Reply:
Gotcha, that was a copy-paste mistake in the comment